<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1144</width>
    <height>747</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>20</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_do">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>510</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Выполнить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_result">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>90</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Результат:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_result_value">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>170</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Значение</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_op1_text">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Операнд 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_op2_text">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>90</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Операнд 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_error">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>160</y>
      <width>391</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Ошибка</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_op1_value">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_op2_value">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>160</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_sum">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>280</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Сложение</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_sub">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>330</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Вычитание</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_mul">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>380</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Умножение</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_div">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>430</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Деление</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_result_ost">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>290</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Остаток</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ost_text">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>220</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Остаток:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1144</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
